core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -258,7 +258,6 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
      * @dev Might not withdraw the full requested amount in order to account for precision loss.
      * @return amountWithdrawn The actual amount of collateral withdrawn.
      */
-    // TODO: Decide whether to fold this functionality into withdraw() method above.
```

Comment:
@mrice32 We had discussed this, I think its still a decent idea to do this, but maybe not given time constraints.

Reply:
I thought we decided not to because it would be confusing to have the collateralAmount arg in this function call.